pdb2pqr/residue.py

```diff
@@ -330,8 +332,7 @@ def reorder(self):
         # Change the list pointer
         self.atoms = templist[:]
 
-    @staticmethod
-    def letter_code():
+    def letter_code(self) -> str:
```

Comment:
All of the inherited classes are not staticmethod, so this definition should change in the parent class.